src/index.tsx

```diff
@@ -8,19 +7,13 @@ import React from 'react';
 import BrowserApp from './domain/app/BrowserApp';
 import { initI18next } from './common/translation/i18n/i18nInit';
 import AppConfig from './domain/app/AppConfig';
+import initSentry from './utils/sentry';
```

Comment:
As you're using default export/import then renaming the file to `initSentry.ts` would make sense, but named export/import would be ok too IMHO.